Lib/http/server.py

```diff
@@ -103,10 +102,9 @@
 import sys
 import time
 import urllib.parse
-import contextlib
-from functools import partial
 
 from http import HTTPStatus
+import http.client
```

Comment:
This import does not need to change, typical import style is to have all `import` first then all `from`.

Reply:
Thanks for the review. First let me point out that *local* imports (of the `http` module and submodules here) should be grouped at the end according to [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports):

> Imports should be grouped in the following order:
> 
> - Standard library imports.
> - Related third party imports.
> - Local application/library specific imports.
> 
> You should put a blank line between each group of imports.

Now about the ordering between `import` and `from` statements, PEP 8 does not say anything, but you are absolutely right, `import` statements are typically first (that is for instance what [`isort`](https://pypi.org/project/isort/) does). Updated, thanks.

Reply:
That doesn’t apply here, we are in the standard library, all imports are section 1.

Reply:
I disagree, take a look at the standard `email` module for instance: https://github.com/python/cpython/blob/main/Lib/email/message.py